### PR TITLE
SM-6708: Update docs for Section 74 list endpoint

### PR DESCRIPTION
**Pull request by pwright08**

### CODE REVIEW

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 1]
```diff
@@ -247,6 +248,17 @@ Retrieves further information for the works provided in the request. This allows
...
 
+#### Section 74s
+{: .govuk-heading-s}
+
+<code>GET /section-74s</code>
+
+Retrieves a list of section 74s which are associated with the authenticated user's organisation.
+{: .govuk-body}
+
+Contractors are required to provide optional swa_code parameter in order to state which promoter they are working on behalf of.
```
Comment (cpaul92) [inline docs/_includes/api-documentation/v3/resource-guide.md:259]: I thought we usually called out the filter parameters too in this section?

Comment (pwright08) [reply docs/_includes/api-documentation/v3/resource-guide.md:259]: Good point, based this one off Section 81s which seem to be missing the filter options also. I'll update both

**Review by RichardEustace — APPROVED** [review]

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 2]
```diff
@@ -247,6 +248,29 @@ Retrieves further information for the works provided in the request. This allows
...
+<ol class="govuk-list govuk-list--bullet">
+  <li><strong>section_74_ha_status</strong>: The Section 74 HA Status i.e. warning issued, resolved</li>
+  <li><strong>issue_date_from</strong>: Date filtering based on the Section 74 issue date</li>
+  <li><strong>issue_date_to</strong>: Date filtering based on the Section 74 issue date</li>
+  <li><strong>sort_column</strong>: The property used to order results by</li>
+  <li><strong>sort_direction</strong>: Ascending/descending</li>
+  <li><strong>query</strong>: Search field for work reference number, section 74 reference number, street information (partial match), USRN, or organisation</li>
+  <li><strong>swa_code</strong>: Optional parameter to be used by contractors only. Used to provide the swa code of the promoter the contractor is working on behalf of</li>
+  <li><strong>geographical_area_reference_number</strong>: An optional array of Geographical Areas that you would like to filter your list by as a HA user</li>
+  <li><strong>offset</strong>: The number of results to skip when returning the resultset, used for pagination</li>
```
Comment (caoimheb) [inline docs/_includes/api-documentation/v3/resource-guide.md:271]:
```suggestion
  <li><strong>offset</strong>: The number of results to skip when returning the result set, used for pagination</li>
```

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 3]
```diff
@@ -255,9 +279,20 @@ external integrators to retrieve additional information for the works returned b
...
 
+<ol class="govuk-list govuk-list--bullet">
+  <li><strong>section_81_status</strong>: The Section 81 Status i.e. issued, accepted</li>
+  <li><strong>section_81_severity</strong>: The Section 81 Severity i.e. high, low</li>
+  <li><strong>sort_column</strong>: The property used to order results by</li>
+  <li><strong>sort_direction</strong>: Ascending/descending</li>
+  <li><strong>query</strong>: Search field for work reference number, section 81 reference number, street information (partial match), USRN, or organisation</li>
+  <li><strong>swa_code</strong>: Optional parameter to be used by contractors only. Used to provide the swa code of the promoter the contractor is working on behalf of</li>
+  <li><strong>geographical_area_reference_number</strong>: An optional array of Geographical Areas that you would like to filter your list by as a HA user</li>
+  <li><strong>offset</strong>: The number of results to skip when returning the resultset, used for pagination</li>
```
Comment (caoimheb) [inline docs/_includes/api-documentation/v3/resource-guide.md:293]:
```suggestion
  <li><strong>offset</strong>: The number of results to skip when returning the result set, used for pagination</li>
```

File: docs/_includes/api-documentation/v3/resource-guide.md (hunk 2 above)
Comment (caoimheb) [inline docs/_includes/api-documentation/v3/resource-guide.md:271]: We don't seem to call out the offset param for any of the other endpoints. It's just included in the pagination section. Maybe inconsistent to add it to these two?

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 4]
```diff
@@ -247,6 +248,29 @@ Retrieves further information for the works provided in the request. This allows
...
+
+<ol class="govuk-list govuk-list--bullet">
+  <li><strong>section_74_ha_status</strong>: The Section 74 HA Status i.e. warning issued, resolved</li>
+  <li><strong>issue_date_from</strong>: Date filtering based on the Section 74 issue date</li>
+  <li><strong>issue_date_to</strong>: Date filtering based on the Section 74 issue date</li>
+  <li><strong>sort_column</strong>: The property used to order results by</li>
+  <li><strong>sort_direction</strong>: Ascending/descending</li>
+  <li><strong>query</strong>: Search field for work reference number, section 74 reference number, street information (partial match), USRN, or organisation</li>
+  <li><strong>swa_code</strong>: Optional parameter to be used by contractors only. Used to provide the swa code of the promoter the contractor is working on behalf of</li>
+  <li><strong>geographical_area_reference_number</strong>: An optional array of Geographical Areas that you would like to filter your list by as a HA user</li>
```
Comment (caoimheb) [inline docs/_includes/api-documentation/v3/resource-guide.md:270]: Do we need to call these out as optional? Are all the params not optional?

**Pull request merged**